mdx-web/src/main/java/com/mx/path/model/mdx/web/controller/ProfilesController.java

```diff
@@ -68,6 +69,23 @@ public final ResponseEntity<?> deleteAddress(@PathVariable("addressId") String a
     return new ResponseEntity<>(createMultiMapForResponse(response.getHeaders()), HttpStatus.NO_CONTENT);
   }
 
+  @RequestMapping(value = "/users/{userId}/profile/challenge_questions", method = RequestMethod.GET)
+  public final ResponseEntity<ChallengeQuestions> getChallengeQuestions() {
+    AccessorResponse<ChallengeQuestions> response = gateway().profiles().challengeQuestions().list();
+    return new ResponseEntity<>(response.getResult().wrapped(), createMultiMapForResponse(response.getHeaders()), HttpStatus.ACCEPTED);
+  }
+
+  @RequestMapping(value = "/users/{userId}/profile/challenge_questions", method = RequestMethod.PUT)
```

Comment:
I think we want `consumes = BaseController.MDX_MEDIA` here as well since the request will be sending a body.

Reply:
Fixed!